.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "files.associations": {
+        "*.spec": "gauge",
+        "*.cpt": "gauge"
+    }
```

Comment:
we're getting rid of gauge. can u remove this?

Reply:
yes, sorry, I thought .vscode was ignored in .gitignore...